PythonTasks/nutritionist/nutritionist.py

```diff
@@ -0,0 +1,74 @@
+
+def height_in_meters(feet, inches):
+    f = float(feet)*0.305
+    i = float(inches)*0.0254
+    meters = f+i
+    return meters
+
+
+def weight_in_kgs(weight):
+    w = float(weight)*0.45
+    return w
+
+
+def bmi_calculation( w, meters):
+    bmi = w / (meters * meters)
+    print("Your BMI is: ",round(bmi,2)," kg/m^2")
```

Comment:
lets not print the results in functions that are supposed to be calculating results.
Lets follow Single Responsibility Principal.